### PR TITLE
(#3879) Add aria-labelledby to components

### LINKED ISSUE #3879: Update components to set aria-labelledby property
### Issue description
Each of the component sections on home and landing pages need to have the aria-labelledby property set. 

> **ESTIMATE** TBD

### Steps to reproduce the issue

1.  
2. 
3. 

### What's the expected result?
-

### What's the actual result?
-

### Additional details / screenshot
- Screenshot

### Related Tickets
* Issue #9999
* Issue #9999

### PR DESCRIPTION
**Pull request by dev-rana-publicis**
Closes #3879
ODE: https://ncigovcdode579.prod.acquia-sites.com/

### CODE REVIEW

**Comment by bennettcc** [comment]
Andy approved this PR on 6/9 - need one more product reviewer then the checkmark should be added

**Review by sarinapadilla — APPROVED** [review]
Looks good to product!

**Comment by dev-rana-publicis** [comment]
Current Changes:

- NCIDS Hero
  - changed from `<section>` to `<div>`
  - no longer has `aria-labelledby` (was on the `section` prior to the `div` change) and no longer has `id` on the Hero's heading
- NCIDS Call to Action Strip (Standalone)
  - changed from `<section>` to `<div>`
  - does not have `aria-labelledby`

- Call to Action Strip (Hero variant & standalone)
  - Bryan noticed (while looking at ACSF on his phone) that the link button text was not the correct color
  - This PR fixes the link button text color

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/paragraph--ncids-2-guide-card-row.html.twig [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {% if (content.field_image_desc_guide_cards['#items']|length > 0) %}
-  <section class="usa-section usa-section--light cgdp-guide-card-row" data-eddl-landing-row>
+  {% set id = paragraph.id() %}
+  {% set entity_type = paragraph.entityTypeID %}
+  {% set titleID = "#{entity_type}-#{id}" %}
+  <section class="usa-section usa-section--light cgdp-guide-card-row" aria-labelledby="{{ titleID }}" data-eddl-landing-row>
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/paragraph--ncids-2-guide-card-row.html.twig:5]: This seems wrong. Nothing has this id.

**Review by bryanpizzillo — CHANGES_REQUESTED** [review]
Can you please look into the thing below?

**Review by arcepaul — APPROVED** [review]

**Pull request merged**